dataprofiler/profilers/profiler_options.py

```diff
@@ -761,3 +761,66 @@ def _validate_helper(self, variable_path='ProfilerOptions'):
             variable_path=variable_path + '.structured_options')
 
         return errors
+
+
+class TextProfilerOptions(BaseOption):
+    def __init__(self):
```

Comment:
properties should be settable on construction with defaults to enabled

Reply:
I think if we want, we have to do it for every other classes. Currently, some of them have, some do not have. Not sure we want to do it in this pR.